features/step_definitions/order_steps.rb

```diff
@@ -14,3 +14,29 @@
 And(/^I fill in Delivery Date with "([^"]*)"$/) do |date|
   fill_in('order_delivery_date', with: date)
 end
+
+
+Given(/^"([^"]*)"'s order contains:$/) do |billing_name, table|
+  @order = Order.find_by(billing_name: billing_name)
+  table.hashes.each do |item|
+    dish = Dish.find_by(name: item[:dish_name])
+    @order.add(dish, dish.price, item[:quantity].to_i)
+  end
+end
+
+Then(/^the total for the order should be "([^"]*)"$/) do |total|
+  @order.reload
+  expect(@order.total).to eq Money.new(total.to_f * 100)
+end
+
+Then /^the subtotal for the cart should be "([^"]*)"$/ do |subtotal|
+  @order = @order || Order.last
```

Comment:
Isn't better to create an `order.id`(corresponding to my comment on line 20?)or something similar?

Reply:
This is just for testing purposes.